app/models/grda_warehouse/hud/project_coc.rb

```diff
@@ -38,6 +38,13 @@ class ProjectCoc < Base
       )
     end
 
+    scope :in_cocs, ->(coc_codes:) do
+      where(
+        pc_t[:CoCCode].in(coc_codes).and(pc_t[:hud_coc_code].eq(nil).or(pc_t[:hud_coc_code].eq(''))).
+        or(pc_t[:hud_coc_code].in(coc_codes)),
+      )
+    end
+
```

Comment:
Is there any functional difference between this and `scope :in_coc, ->(coc_code:)`?  They look identical, other than the use of plurals.

Reply:
it already does.  `coc_code = Array(coc_code)` handles that.